resources/views/store/create_registration.blade.php

```diff
@@ -1,220 +1,34 @@
-                    var displayVerified = verified === 1 ? "checked" : null;
-                    var displayMonths = moment().diff(dob, 'months') % 12;
-                    var displayYears = moment().diff(dob, 'years');
-
-                    // Create and append new style columns
-                    var ageColumn = '<td class="age-col">' + displayYears + ' yr, ' + displayMonths + ' mo</td>';
-                    var dobColumn = '<td class="dob-col"><input name="children[' + childKey + '][dob]" type="hidden" value="' + dob + '" >' + dateObj + '</td>';
-                    var idColumn = '<td class="verified-col relative"><input type="checkbox" class="styled-checkbox inline-dob" name="children[' + childKey + '][verified]" id="child' + childKey + '" ' + displayVerified + ' value="' + verified + '"><label for="child' + childKey + '"><span class="visually-hidden">Toggle ID checked</span></label>' + '</td>';
-                    var removeColumn = '<td class="remove-col"><button type="button" class="remove_date_field"><i class="fa fa-minus" aria-hidden="true"></i></button></td>';
-
-                    $(this).append(ageColumn);
-                    $(this).append(dobColumn);
-                    $(this).append(idColumn);
-                    $(this).append(removeColumn);
-                });
-
-                $(el).on("click", ".remove_field", function (e) {
-                    e.preventDefault();
-                    $(this).closest('tr').remove();
-                    fields--;
-                })
-            }
-        );
+                @include('store.partials.voucher_collectorsSP')
+                @include('store.partials.householdSP')
+                @include('store.partials.other_infoSP')
+            </form>
+        </div>
+        @else
+        <div class="content">
+            <form action="{{ URL::route("store.registration.store") }}" method="post" class="full-height">
+            {!! csrf_field() !!}
+                @include('store.partials.voucher_collectors')
+                @include('store.partials.family')
+                @include('store.partials.other_info')
+            </form>
+        </div>
+        @endif
 
-        // If enter is pressed, keyboard is hidden on iPad and form submit is disabled
-        $('#carer').on('keyup keypress', function(e) {
-            if(e.which === 13) {
-                e.preventDefault();
-                document.activeElement.blur();
-                $("input").blur();
-                return false;
-            }
-        });
-        //remove invalid class when input is selected/tabbed to
-        $('#privacy-statement, #carer').on('click focus', function() {
-            $(this).removeClass("invalid");
-            // Remove relevent error message
-            var spanclass = $(this)[0].id + '-span';
-            $('#' + spanclass).addClass('collapsed');
-        });
-    </script>
+<script src="{{ asset('store/js/create_registration.js') }}"></script>
```

Comment:
look at how mix is doing the copying and mangling of javascript into `public/`

Reply:
As discussed, we will be leaving this well alone for now!